src/components/remotecontrol/remotecontrol.js

```diff
@@ -221,6 +263,12 @@ define(["browser", "datetime", "backdrop", "libraryBrowser", "listView", "imageL
                 context.querySelector(".navigationSection").classList.add("hide");
             }
 
+            if ((-1 != supportedCommands.indexOf("DisplayMessage") || -1 != supportedCommands.indexOf("SendString") || -1 != supportedCommands.indexOf("Select")) && !currentPlayer.isLocalPlayer) {
```

Comment:
Use `.includes()` instead of `.indexOf`

Reply:
I'm not sure here. Could you give me a example?

Reply:
@MrTimscampi speaking of which, I dislike literal arrays on the left side of a conditional. Can we avoid those?

Reply:
@samuel9554 Something like this:
```suggestion
if ((supportedCommands.includes("DisplayMessage") ||
    supportedCommands.includes("SendString") ||
    supportedCommands.includes("Select")) &&
    !currentPlayer.isLocalPlayer) {
```

@dkanada If you mean this:
```javascript
if ([0, 1, 2, 3].includes(2)) {
```
Then yeah, the `[0, 1, 2, 3]` in that case should be moved to a constant assignment before the condition.

Reply:
Resolve in last commit